app/main/views/marketplace.py

```diff
@@ -118,7 +116,11 @@ def list_opportunities(framework_framework):
 
     api_result = form.get_briefs()
 
-    briefs = api_result["briefs"]
+    briefs = [{
+        "data": brief,
+        "content": content_loader.get_builder(brief['frameworkSlug'], 'display_brief').filter(brief)
```

Comment:
I'm relatively sure (about 95%) that `get_builder` is deprecated and we should use `get_manifest` everywhere these days.